frontend/src/lib/components/instance-views/views/views/openai-chat-markdown/AssistantBlock.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+	import Markdown from '$lib/components/general/Markdown.svelte';
+	import purify from 'isomorphic-dompurify';
+	import { parse } from 'marked';
+
+	export let input: string;
+	export let output = false;
+
+	let renderedInput = purify.sanitize(parse(input));
+</script>
+
+<div class="my-1 flex items-end relative z-0">
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 640 512"
+		class={`${output ? 'fill-primary' : 'fill-grey-darker'} w-6 mr-2.5 mt-2`}
+	>
+		<path
+			d="M320 0c17.7 0 32 14.3 32 32V96H472c39.8 0 72 32.2 72 72V440c0 39.8-32.2 72-72 72H168c-39.8 0-72-32.2-72-72V168c0-39.8 32.2-72 72-72H288V32c0-17.7 14.3-32 32-32zM208 384c-8.8 0-16 7.2-16 16s7.2 16 16 16h32c8.8 0 16-7.2 16-16s-7.2-16-16-16H208zm96 0c-8.8 0-16 7.2-16 16s7.2 16 16 16h32c8.8 0 16-7.2 16-16s-7.2-16-16-16H304zm96 0c-8.8 0-16 7.2-16 16s7.2 16 16 16h32c8.8 0 16-7.2 16-16s-7.2-16-16-16H400zM264 256a40 40 0 1 0 -80 0 40 40 0 1 0 80 0zm152 40a40 40 0 1 0 0-80 40 40 0 1 0 0 80zM48 224H64V416H48c-26.5 0-48-21.5-48-48V272c0-26.5 21.5-48 48-48zm544 0c26.5 0 48 21.5 48 48v96c0 26.5-21.5 48-48 48H576V224h16z"
```

Comment:
I didn't know what the semantics of this are. I don't know if it's standard to add comments in Svelte, but if so we could add one here.

Reply:
Hmm like comment that this is an SVG path? Not sure I understand.

Reply:
Ah or comment what type of icon this will render as?

Reply:
Yeah, a comment about what type of icon this renders as would be helpful!

Reply:
Will do in a separate PR. I think we can change most of them for material icons now.